.github/workflows/feature-branch-image.yml

```diff
@@ -0,0 +1,31 @@
+---
+name: Build and Push Feature Branch Custom Catalog
+env:
+  BUNDLE_IMG: quay.io/${{ secrets.QUAY_ORG }}/opendatahub-operator:${{ github.head_ref }}
+  CATALOG_IMG: quay.io/${{ secrets.QUAY_ORG }}/opendatahub-operator-catalog:${{ github.head_ref }}
+on:
+  pull_request:
+    branches:
+      - "feature-**"
+permissions:
+  contents: read
```

Comment:
Hi @CFSNM , IIRC the contents have `read` permissions by default? 🤔

Reply:
yes, it seems to be working without the read permissions. Removing them